### ISSUE TITLE
Invalid country_code value when no argument is supplied

### ISSUE THREAD

**Issue by InfamousSYN** [InfamousSYN/rogue#38]
When deploying an AP using the following command, an exception is raised reporting an invalid country_code value.
```
sudo python3 /opt/rogue/rogue.py -i wlan0 --auth wpa-personal --wpa 2 --wpa-passphrase "test test" --internet --preset-profile wifi4 --channel-randomiser
```

The full stack trace below:
```
──(kali㉿kali)-[/opt/rogue]
└─$ sudo python3 /opt/rogue/rogue.py -i wlan0 --auth wpa-personal --wpa 2 --wpa-passphrase "test test" --internet --preset-profile wifi4 --channel-randomiser
[sudo] password for kali: 
[*] Launching the rogue toolkit v3.0.3
[-]
[-] Randomised channel selection is superseding ACS
[-]   Channel 1 was selected
[-] Launching rogue at: 2023-01-12 20:30:51.615436

[*] Reticulating radio frequency splines...

100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:01<00:00,  1.00s/it]

[+] Creating hostapd-wpe.confg file: /opt/rogue/tmp/hostapd-wpe.conf
[+] Creating /etc/default/isc-dhcp-server file: /etc/default/isc-dhcp-server
[+] Creating dhcpd.conf file: /etc/dhcp/dhcpd.conf

[*] Starting isc-dhcp-server service.

100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:04<00:00,  1.00s/it]

[*] Launching hostapd-wpe
Line 104: Invalid country_code '0'
1 errors found in configuration file '/opt/rogue/tmp/hostapd-wpe.conf'
Failed to set up interface with /opt/rogue/tmp/hostapd-wpe.conf
Failed to initialize interface
[*] Enabling IP forwarding
Press enter to quit...
```

**Comment by Xwaasaa** [comment]
I have the same issue... Found a fix ?

**Comment by sinfulz** [comment]
> I have the same issue... Found a fix ?

The interim fix seems to be specifying a country with `--country AU`.

**Comment by InfamousSYN** [comment]
Resolved in [commit 04b9b1a](https://github.com/InfamousSYN/rogue/commit/04b9b1a992ad0135f2cebe386e6ce824116148be), with the fixed pushed to master branch in [PR 41](https://github.com/InfamousSYN/rogue/pull/41)
